build/_k-test.shade

```diff
@@ -3,6 +3,7 @@ use import="Environment"
 
 default NO_PARALLEL_TEST_PROJECTS='${E("NO_PARALLEL_TEST_PROJECTS")}'
 default KOREBUILD_TEST_DNXCORE='${E("KOREBUILD_TEST_DNXCORE")}'
+default CORECLR_RUNTIME_VERSION='${E("CORECLR_RUNTIME_VERSION")}'
```

Comment:
I actually think it is not in dnvm. Sake runs:
`cmd /C "C:\Source\dnx\packages\KoreBuild\build\dnvm" run default -runtime coreclr test` 
and the exitcode is lost. 
When I run:
`& "C:\Source\dnx\packages\KoreBuild\build\dnvm" run default -runtime coreclr test`
the exit code is preserved.
I also confirmed that dnvm in the KoreBuild have the fixes for  aspnet/dnvm#428